autobahn/websocket/protocol.py

```diff
@@ -2146,11 +2146,11 @@ def sendCloseFrame(self, code=None, reasonUtf8=None, isReply=False):
         """
         if self.state == WebSocketProtocol.STATE_CLOSING:
             if self.debugCodePaths:
-                self.factory._log("ignoring sendCloseFrame since connection is closing")
```

Comment:
More that people might not have debug turned off, and we'll end up inadvertantly filling up logs.

The PassthroughLogger will gain a new later function (`.trace()`) which will then call debug. This patch just gets the groundwork in.